Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -142,8 +142,14 @@ namespace Multiplayer
         }
 
         // Start the configured server if it's available
+        AZStd::string projectPath(AZ::Utils::GetProjectPath().c_str());
+        AZStd::replace(projectPath.begin(), projectPath.end(), AZ::IO::WindowsPathSeparator, AZ::IO::PosixPathSeparator);
         AzFramework::ProcessLauncher::ProcessLaunchInfo processLaunchInfo;
-        processLaunchInfo.m_commandlineParameters = AZStd::string::format("\"%s\" --editorsv_isDedicated true", serverPath.c_str());
+        processLaunchInfo.m_commandlineParameters = AZStd::string::format(
+            R"("%s" --project-path "%s" --editorsv_isDedicated true --sv_defaultPlayerSpawnAsset "%s")",
```

Comment:
How does this affect spawning the Editor prefab? The idea here is that Ctrl+G generates a spawnable based on the current contents of the level and that is serialized and transmitted to the server for it to spawn. This seems like it would start the server with the player prefab instead.

Reply:
Yes, the current functionality is the server spawns one sv_defaultPlayerSpawnAsset for every client connected.  By default this is "prefabs/player.network.spawnable".  This allows the user to tell the launched server whatever our  editor's sv_defaultPlayerSpawnAsset is set.

Reply:
Ah okay that's a good change then.